tes-adapter/src/main/java/com/epam/pipeline/tesadapter/service/TesTaskService.java

```diff
@@ -0,0 +1,7 @@
+package com.epam.pipeline.tesadapter.service;
+
+import org.springframework.stereotype.Service;
+
+@Service
```

Comment:
You need to provide an implementation for this interface or application won't start